### ISSUE TITLE
Cryptonightv7

### ISSUE THREAD

**Issue by GuyTechie** [Sniffdog/Sniffdogminer#214]
cpuminer-opt 3.8.8 now supports cryptonightv7.  Can you add that to the script?  MPH has that coin for sure.  Not sure about the other pools.

**Comment by Sniffdog** [comment]
done and in master

**Comment by GuyTechie** [comment]
I took a look, but didn't see any references to cryptonightv7 being added or enabled.  I did see you updated cpuminer-opt to download the newest version (3.8.8).

I tried adding cryptonightv7 manually myself, but I'm not sure if I fully understand how this script works.  I added it to Algorithms.txt, the BAT file, and the miner file.  It should show up in the dashboard console for benching, but there's no pool to hash/bench.

**Comment by Sniffdog** [comment]
Yeah my apologies.... I have been busy with shutdowns this year (very busy)

Added to JayDDee miner and Algo.txt... not sure if/when zergpool will add

**Comment by GuyTechie** [comment]
No problem.  FYI, MPH has it though.  Hopefully it will work with the new commits.

**Comment by GuyTechie** [comment]
Forgot to mention I still couldn't get Cryptonightv7 to work
